syntax = "proto3";

package Proxy;
option cc_generic_services = true ;

message ProxyRequest
{
    bytes type = 1;
    bytes request_msg = 2;
}

message ProxyResponse
{
    bytes type = 1;
    bytes response_msg = 2;
}
